### PR TITLE
Fix for Regen and Blink with SCH strategems.

### PR DESCRIPTION
**Pull request by Karumac**
I found the reason why Perpetuation would cause Regen to freeze up the program. The line that actually uses Perpetuation didn't exist so the thing would loop trying to cast but never actually sending the command to XI.

### CODE REVIEW

File: Form1.cs [hunk 1]
```diff
@@ -6150,6 +6162,7 @@ private async void actionTimer_TickAsync ( object sender , EventArgs e )
 
                                    if ( Form2.config.Perpetuance && Form2.config.regenPerpetuance && currentSCHCharges > 0 && HasAbility ( "Perpetuance" ) && !plStatusCheck ( StatusEffect.Perpetuance ) )
                                    {
+                                        JobAbility_Wait("Regen, Perpetuance", "Perpetuance");
```
Comment (Karumac) [inline Form1.cs]: This line I just copied from another spell's entry. For whatever reason, it just didn't exist.

File: Form1.cs [hunk 2]
```diff
@@ -6093,7 +6093,19 @@ private async void actionTimer_TickAsync ( object sender , EventArgs e )
...
+                                        JobAbility_Wait("Blink, Accession", "Accession");
+                                        return;
+                                    }
+
+                                    if (Form2.config.Perpetuance && Form2.config.blinkPerpetuance && currentSCHCharges > 0 && HasAbility("Perpetuance") && !plStatusCheck(StatusEffect.Perpetuance))
+                                    {
+                                        JobAbility_Wait("Blink, Perpetuance", "Perpetuance");
+                                        return;
+                                    }
+
```
Comment (Karumac) [inline Form1.cs]: These lines I just copied from another entry. For whatever reason, even though blink had perp and acces buttons, it had no code to check those buttons.

**Comment by DanielHazzard** [comment]
Thank you for finding the error, I can't push that release as it's placed incorrectly, the way you've donev it would make it cast the spell then Accession/Perpetuance.

That said I'll push a fix by the end of today.

**Comment by DanielHazzard** [comment]
Pushed a fix, thank you for the help once more, closing.

**Pull request closed without merging**